program.go

```diff
@@ -33,6 +34,8 @@ var (
 	userSubscriptionID uuid.UUID
 	userTenantID       uuid.UUID
 	environment        = azure.PublicCloud
+
+	sampleUA = fmt.Sprintf("sample/0005/%s", utils.GetCommit())
```

Comment:
Honestly, I don't like the "sample/0005" tag. I think it introduces a level of indirection that we don't need. What's wrong with, `sampleUA = fmt.Sprintf("github.com/Azure-Samples/arm-compute-go-vm-extensions@%s", utils.GetCommit())`